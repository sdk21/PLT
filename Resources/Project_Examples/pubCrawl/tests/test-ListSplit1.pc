sentence = "this is a sentence that has words";

result = List.split(sentence, ' ');

print(result[0]);
print(result[1]);
print(result[2]);
print(result[3]);
print(result[4]);
print(result[5]);
print(result[6]);
