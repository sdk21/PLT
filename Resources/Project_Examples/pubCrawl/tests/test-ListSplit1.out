this
is
a
sentence
that
has
words
